packages/wallet/src/views/Assets.jsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) Kernel
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { ethers } from 'ethers'
+import { useEffect, useState } from 'react'
+
+import Page from 'components/Page'
+
+const WALLET_STORE_VERSION = '1'
+
+const getItem = (k) => JSON.parse(localStorage.getItem(k))
+
+const loadWallet = () => {
+  const wallet = getItem('wallet')
+  if (wallet.version !== WALLET_STORE_VERSION) {
+    throw new Error('unsupported wallet')
+  }
+  return wallet
+}
+
+const portalCards = [
+  {
+    title: `Lootbag`,
+    description: `Claim some test ETH to start`,
+    url: '/lootbag',
+  },
+  {
+    title: `Explore`,
+    description: `Understand how transactions work`,
+    url: '/explore',
+  },
+  {
+    title: `Deploy`,
+    description: `Create your own token now`,
+    url: '/deploy',
+  },
+  {
+    title: `Mint`,
+    description: `Create an NFT in one go`,
+    url: '/mint',
+  },
+  {
+    title: `Address Book`,
+    description: `Get your friends involved`,
+    url: '/address-book',
+  },
+  {
+    title: `Feature Requests`,
+    description: `Learn how to improve this with us`,
+    url: '/feature-requests',
+  },
+  {
+    title: `Contracts`,
+    description: `Explore all the best contracts`,
+    url: '/contracts',
+  },
+  {
+    title: `Develop`,
+    description: `Write your own code!`,
+    url: '/develop',
+  },
+]
```

Comment:
I know I added an App.config, but I feel like this content makes sense in this component's file, so you can change it all in one place.